radixdlt-core/radixdlt/src/test/java/com/radixdlt/client/store/berkeley/BalanceEntryTest.java

```diff
@@ -29,7 +29,7 @@
 
 public class BalanceEntryTest {
 	private static final RadixAddress ADDRESS = RadixAddress.from("JH1P8f3znbyrDj8F4RWpix7hRkgxqHjdW2fNnKpR3v6ufXnknor");
-	private static final Rri TOKEN_RRI = Rri.of(ADDRESS.getPublicKey(), "xrd");
+	private static final REAddr TOKEN_RRI = REAddr.ofHashedKey(ADDRESS.getPublicKey(), "xrd");
```

Comment:
@talekhinezh what is this? This is creating "xrd" rri? I thought that was. a"system rri" and thus not using any public key at all? Now Im confused....